internal/server/v0/internalbio/golden/get_bio_page_data/yeast.json

```diff
@@ -27,9 +27,6 @@
                           "property": "value",
                           "direction": "DIRECTION_OUT",
                           "nodes": [
-                            {
-                              "value": "0.35"
-                            },
```

Comment:
@shifucun is this deletion expected?

Reply:
Yes, this is a bug that I fixed in https://github.com/datacommonsorg/mixer/commit/830a4f19f7ddb3c530128432c00ac23b7e703c29. If you sync, the diff should go away.

Reply:
Thanks - just saw your fix too.